design/one-pager-stack-relationship-labels.md

```diff
@@ -0,0 +1,49 @@
+# Stack Parent/Child Relationship Label Spec
+
+- Owner: Steven Rathbauer (@rathpc)
+- Reviewers: Crossplane Maintainers
+- Status: Draft
+
+## Proposal
+
+[crossplaneio/crossplane#934](https://github.com/crossplaneio/crossplane/issues/934)
+
+## Problem
+
+There is currently no easy way to get a list of all resources that are owned by a parent resource. We can leverage CRD labels to make this easier. As long as we have a standard format for doing so.
+
+Per the proposal, it seems that the clearest example of what I am referring to is shown on this page https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels within the `app.kubernetes.io/part-of` label.
+
+As long as we define this label on all children CRD's, we should then be able to query all children of a given parent name using those labels.
+
+## Design
+
+### Example `stack.yaml`
+
+```yaml
+apiVersion: stacks.crossplane.io/v1alpha1
+kind: Stack
+metadata:
+ name: example-app-name
+...
+```
+
+### Example `crd.yaml`
+
+```yaml
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+    core.crossplane.io/part-of: "example-app-name"
+  name: mytypes.samples.crossplane.io
+...
+```
+
+As per the above two file example snippets, as long as the `core.crossplane.io/part-of` label is defined on children CRD's to the value of the stacks name, then we can use that label to look up all children of a given parent.
```

Comment:
If I'm understanding the [recommended labels document that was linked earlier](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels), it would make the most sense to use the labels as described in that document. So, `app.kubernetes.io/part-of` instead of `core.crossplane.io/part-of`. The reason to do this would be so that other tools can use the same labels.

Reply:
That works for me!